kubernetes/client/api_client.py

```diff
@@ -343,6 +344,13 @@ def request(self, method, url, query_params=None, headers=None,
         """
         Makes the HTTP request using RESTClient.
         """
+        if url.endswith('/exec') and method == "GET":
+            return ws_client.GET(self.config,
```

Comment:
I maybe not familiar enough with exec command, but does it let interactive connection with the host? I mean can a script send something, then read the result, then decide what next command to send, etc.? In that case, should we return some kind of client allows that instead of command result?